modules/compiler/source/base/source/module.cpp

```diff
@@ -238,7 +238,12 @@ static bool loadKernelAPIHeader(clang::CompilerInstance &compiler,
   // stored inside the PCH file.
   llvm::BitstreamCursor &Cursor = moduleFile->InputFilesCursor;
   clang::SavedStreamPosition SavedPosition(Cursor);
-  if (Cursor.JumpToBit(moduleFile->InputFileOffsets[0])) {
+  uint64_t Base = 0;
```

Comment:
I would expect this to trigger a warning 'warning "Base" set but not used' when building against LLVM 18, depending on the host compiler. Or is that a warning we suppress?

Reply:
Using version 11 of... Some compiler (seriously, `--version` should include the compiler name - I assume it's GCC based on the license), I get no warnings. Not sure why this would trigger a warning, Base is being used in the JumpToBit call below.

Reply:
This value being assigned isn't used: it's immediately overwritten with `moduleFile->InputFilesOffsetBase`. Hold on, let me try to get something together on Godbolt that demonstrates the warning and we'll see why it does or doesn't apply here.

Reply:
Hmm, I must be misremembering, I can only find compiler versions that warn about the whole variable never being used, not one specific assignment. Alright then, if compilers are happy with it, I have no reason to complain.